app/rpc/namespaces/eth/filters/utils.go

```diff
@@ -101,3 +105,19 @@ func returnLogs(logs []*ethtypes.Log) []*ethtypes.Log {
 	}
 	return logs
 }
+
+//gasPrice: to get "minimum-gas-prices" config or to get ethermint.DefaultGasPrice
+func ParseGasPrice() *hexutil.Big {
```

Comment:
move this function to file `eth/api.go`, cause there was no relationship between gasprice and filter